### ISSUE TITLE
Request: Inbox Copy/Share Option

### ISSUE THREAD

**Issue by Rbagman** [QuantumBadger/RedReader#429]
Being able to copy/share ibox mail to clipboard would be helpful.

**Comment by QuantumBadger** [comment]
Agreed -- it would be nice to have a full menu of options for inbox messages, like for comments/posts.
